ConvivaIntegration/components/conviva/ConvivaAnalyticsTask.brs

```diff
@@ -326,7 +326,7 @@ function getAd(mediaId)
 end function
 
 function mapAdTrackingMode(mode)
```

Comment:
This method is no obsolete:

`if 1 = 1 then return 1`

Just remove the method and use the `m.top.config.adTrackingMode`